src/sparseml/export.py

```diff
@@ -0,0 +1,185 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from pathlib import Path
+from typing import Any, Callable, List, Optional, Union
+
+from pydantic import BaseModel, Field
+
+from sparseml.pytorch.opset import TORCH_DEFAULT_ONNX_OPSET
+from sparsezoo.utils.registry import RegistryMixin
+
+
+_LOGGER = logging.getLogger(__name__)
+AVAILABLE_DEPLOYMENT_TARGETS = ["deepsparse", "onnxruntime"]
+
+
+class SparseMLIntegrationRegistry(RegistryMixin):
```

Comment:
note - we can use mixin directly on the registered class if we want to (as an additional parent class)